coalib/bears/Bear.py

```diff
@@ -217,3 +228,15 @@ def get_config_dir(self):
         :return: Directory of the config file
         """
         return get_config_directory(self.section)
+
+    @classproperty
+    def data_dir(cls):
+        """
+        Returns a directory that may be used by the bear to store stuff. Every
+        bear has an own directory dependent on his name.
+        """
+        data_dir = abspath(join(user_data_dir('coala-bears'), cls.name))
+
+        if not isdir(data_dir):  # pragma: no cover
+            makedirs(data_dir)
```

Comment:
I think you can use `exist_ok=True` as a parameter. Then you don't need the if :) (and also no pragma no cover)